packages/components/src/color-palette/index.tsx

```diff
@@ -148,9 +157,9 @@ export function CustomColorPickerDropdown( {
 }
 
 export const extractColorNameFromCurrentValue = (
-	currentValue,
```

Comment:
`value` is passed to this function, and it [can be undefined](https://github.com/WordPress/gutenberg/blob/6ffdde0670c687dee4b0569853ff76ef656961f1/packages/components/src/color-palette/README.md#value)